src/main/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendor.java

```diff
@@ -43,6 +43,7 @@ public class PaymentWorksVendor extends PersistableBusinessObjectBase implements
     private String requestingCompanyLegalFirstName;
     private String requestingCompanyLegalLastName;
     private String requestingCompanyName;
+    private String requestingCompanyLegalNameForProcessing;
```

Comment:
I'm having trouble finding areas in the updated code where this new PaymentWorks property is being used. Can it be removed?

Reply:
After the opencsv version upgrade, writing to the csv file would not work for this particular field by just calling the getRequestingCompanyLegalNameForProcessing method, I had to explicitly also add this property in order for it to work.

Reply:
After looking through the updated OpenCSV code further, it is indeed the case that the newer OpenCSV versions are expecting the instance field to be present, even if the library relies upon the getter/setter methods to access its value. I don't see an easy way to customize the setup to avoid this, so I'm okay with having the unused field as a temporary workaround.

It would still be helpful to include a comment indicating why this unused field is needed. We may also want a future user story to investigate cleaning this up.

Reply:
Comment added